backend/db/migrations/V1.0.15__report_locking.sql

```diff
@@ -0,0 +1,4 @@
+SET search_path TO pay_transparency;
+
+alter table pay_transparency_report add column unlocked boolean not null default false;
```

Comment:
please rename to is_unlocked - as the other booleans in the DB are named this way

Reply:
Also refactor the variables below accordingly